jobs/jobs.go

```diff
@@ -4,71 +4,111 @@
 package jobs
 
 import (
-	"sync"
+	"context"
+	"time"
 
 	l4g "github.com/alecthomas/log4go"
-	ejobs "github.com/mattermost/platform/einterfaces/jobs"
 	"github.com/mattermost/platform/model"
 	"github.com/mattermost/platform/store"
-	"github.com/mattermost/platform/utils"
 )
 
-type Jobs struct {
-	startOnce sync.Once
+func CreateJob(store store.Store, jobType string, jobData map[string]interface{}) (*model.Job, *model.AppError) {
```

Comment:
It's how the jobserver was originally. I think that the underlying reason is we can't use `app.Srv.Store` because this code can't depend on the `app` package.

However, I think we could instead use the JobServer.Store in a similar fashion, if we ensure that the JobServer object is initialised both when running from the `platform` entrypoint, or the `jobserver` one. @hmhealey can you see any reason why this wouldn't work?